services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/gdpr/KortistoGdprServiceITest.kt

```diff
@@ -344,14 +344,11 @@ class KortistoGdprServiceITest(
 
         @Test
         fun `deletes hanke with applications when there are no other users`() {
-            lateinit var kayttaja: HankekayttajaEntity
-            val hanke =
-                hankeFactory.builder(USERNAME).withHankealue().saveWithYhteystiedot {
-                    kayttaja = hankeKayttajaService.getKayttajaByUserId(hankeEntity.id, USERNAME)!!
-                    omistaja(kayttaja)
-                }
+            val hanke = hankeFactory.builder(USERNAME).withHankealue().saveEntity()
+            val founder = hankeKayttajaService.getKayttajaByUserId(hanke.id, USERNAME)!!
```

Comment:
The way I see the language question is: We should use Finnish for the domain terminology since the Finnish terms are the most accurate. Translating them (by developers) will lead to some inconsistencies between naming in the code and what's shown to users in the UI.

We can use English for everything else because Finnish grammar doesn't lend itself easily to being used in code.

In this context, I don't see founder as a domain-specific term, it's just a user who happened to found the hanke.

Reply:
I somewhat desegree because we use the term "perustaja" or "hankkeen perustaja" when we discuss hanke related things. But, I will not object using "founder".